app/container.php

```diff
@@ -56,12 +56,20 @@
     return new TmpDirectoryCreator($c['filesystem']);
 };
 
-$c['coverage.dir.phpunit'] = function (Container $c): string {
-    return $c['tmp.dir'] . '/' . CodeCoverageData::PHP_UNIT_COVERAGE_DIR;
+$c['tmp.dir'] = function (Container $c): string {
+    return $c['tmp.dir.creator']->createAndGet($c['infection.config']->getTmpDir());
 };
 
-$c['coverage.dir.phpspec'] = function (Container $c): string {
-    return $c['tmp.dir'] . '/' . CodeCoverageData::PHP_SPEC_COVERAGE_DIR;
+$c['coverage.dir.phpunit'] = function (Container $c) {
```

Comment:
is this service using elsewhere? can't find any usage

p.s. the same for phpspec

Reply:
https://github.com/infection/infection/blob/master/src/Command/InfectionCommand.php#L278